test/runner/setup/remote.sh

```diff
@@ -40,6 +41,7 @@ elif [ "${platform}" = "rhel" ]; then
     done
 
     pip --version 2>/dev/null || curl --silent --show-error https://bootstrap.pypa.io/get-pip.py | python
+    pip install passlib
```

Comment:
Is the version in RHEL extras insufficient?

Reply:
I didn't have any luck finding it in the yum repos our RHEL test instance. I figured this was the path of least resistance rather than adding repos (though extras may be there just disabled, I don't remember).

Reply:
Installation of dependencies for tests should be done as part of the tests. That way we avoid the overhead of installing dependencies when they are not needed.

If the dependency is only needed for a single test and does not need to be installed before ansible-playbook runs, then simply install it as the first task in the test target role. Otherwise, put it into a `setup_` target.

Tests can then reference the setup target using ansible meta dependencies, or if needed before ansible-playbook runs, using the `setup/once/{target_name}` or `setup/always/{target_name}` aliases.

For passlib specifically, you'll want to create `setup_passlib` and add the `setup/once/setup_passlib` alias to the `test/integration/targets/vars_prompt/aliases` file.